docs/release-notes/1.7.0.md

```diff
@@ -6,3 +6,4 @@
 - Now, Istio Operator does not restart Pods with Istio Sidecar, which contain custom image annotations. See the issue [#698](https://github.com/kyma-project/istio/issues/698) and [Istio Resource Annotations](https://istio.io/latest/docs/reference/config/annotations/#SidecarProxyImage) for more details.
 - Change Istio Ingress Gateway's scaling to be based only on CPU utilization. This adjustment ensures that the scaling is more responsive to traffic changes, as the Istio Ingress Gateway memory utilization is not a good indicator of the traffic load.
 - Set the default number of Istio Ingress Gateway replicas in smaller clusters to `1`.
+- Set Istio CR status to Warning instead of Error when the Istio version update is not allowed [#855](https://github.com/kyma-project/istio/pull/855).
```

Comment:
```suggestion
- Set Istio CR status to `Warning` instead of `Error` when the Istio version update is not allowed [#855](https://github.com/kyma-project/istio/pull/855).
```